### PR TITLE
bump(main/teleport-tsh): 13.3.8

### PR DESCRIPTION
**Pull request by Shareed2k**

### CODE REVIEW

File: packages/teleport-tsh/build.sh [hunk 1]
```diff
@@ -17,6 +17,7 @@ termux_step_make_install() {
 	cd teleport
 
 	git checkout "v$TERMUX_PKG_VERSION"
+	curl -s https://gist.githubusercontent.com/Shareed2k/f0bdab37f8cb93e320a4e09990b9bb71/raw/93b7f434d952f09d9cb70c43c2b5395dbc384310/metadata_linux.go.patch | git apply
```
Comment (licy183) [inline packages/teleport-tsh/build.sh:20]: Thanks for your contribution. Please add this patch file to git repository.

**Review by licy183 — COMMENTED** [review]
Please squash the commits and rewrite the commit message with `bump(main/teleport-tsh): 13.3.8`. See https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#commit-guidelines.

**Review by licy183 — APPROVED** [review]
LGTM. Thanks!

File: packages/teleport-tsh/build.sh [hunk 2]
```diff
@@ -2,21 +2,20 @@ TERMUX_PKG_HOMEPAGE=https://github.com/gravitational/teleport
 TERMUX_PKG_DESCRIPTION="Secure Access for Developers that doesn't get in the way"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=11.0.3
-TERMUX_PKG_REVISION=3
-TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_VERSION=13.3.8
+TERMUX_PKG_SRCURL=https://github.com/gravitational/teleport/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=1eb2471c144c480abe7effa0e90e921e050ab6964f06ab51381f22c119871967
+TERMUX_PKG_REVISION=4
```
[the remark this answers is not in the document]
Comment (Shareed2k) [reply packages/teleport-tsh/build.sh:8]: i can remove `TERMUX_PKG_REVISION` or no need to bump it ?

[the remark this answers is not in the document]
Comment (truboxl) [reply packages/teleport-tsh/build.sh]: Bumping TERMUX_PKG_REVISION is needed only when you made changes to the package but the TERMUX_PKG_VERSION remains the same.

When upgrading packaging, changing TERMUX_PKG_VERSION is suffice. TERMUX_PKG_REVISION should be removed.

[the remark this answers is not in the document]
Comment (Shareed2k) [reply packages/teleport-tsh/build.sh]: Done

**Review by truboxl — APPROVED** [review]
Thanks

**Pull request merged**